library/build.gradle.kts

```diff
@@ -70,6 +70,10 @@ android {
     buildFeatures {
         buildConfig = true
     }
+    defaultConfig {
+        val consumerFile = file("consumer-rules.pro")
```

Comment:
This doesn't match with the `consumer-rules.pro` path.
[`file()`](https://docs.gradle.org/8.10/javadoc/org/gradle/api/Project.html#file(java.lang.Object)) refers to **this project**.
So `library/`
But the consumer-rules.pto is in `library/src/androidMani/consumer-rules.pro`.